site/developer-tools.html

```diff
@@ -479,7 +479,7 @@ <h4>Solving a binary incompatibility</h4>
 <p>For the problem described above, we might add the following:</p>
 
 <figure class="highlight"><pre><code class="language-scala" data-lang="scala"><span class="c1">// [SPARK-zz][CORE] Fix an issue
-</span><span class="nv">ProblemFilters</span><span class="o">.</span><span class="py">exclude</span><span class="o">[</span><span class="kt">DirectMissingMethodProblem</span><span class="o">](</span><span class="s">"org.apache.spark.SomeClass.this"</span><span class="o">)</span></code></pre></figure>
+</span><span class="nc">ProblemFilters</span><span class="o">.</span><span class="n">exclude</span><span class="o">[</span><span class="kt">DirectMissingMethodProblem</span><span class="o">](</span><span class="s">"org.apache.spark.SomeClass.this"</span><span class="o">)</span></code></pre></figure>
```

Comment:
Just checking, are you using the latest jekyll? I know this caused variations on classes like this. I'm not sure off the top of my head whether this is updating or reverting what the latest one does.

Reply:
It is generated by `v3.8.5`. I updated jekyll to `v4.0.0`: https://github.com/apache/spark-website/pull/245/commits/27dac9e39da700fa22b8e461b7ea21f3249cf327